converter/internal/common/diagnostics.go

```diff
@@ -0,0 +1,38 @@
+// Package common exposes error types used throughout converter and a method to
+// pretty-print them to the screen.
+package common
```

Comment:
I think this just needs to be in its own package since you effectively have to alias everything (and you unfortunately can't put it in converter right now because it would be an import cycle).

Reply:
Though IMO it's a code smell that we have to put it in its own package to avoid circular dependencies, I don't think internal package layout should impact the API